spec/intl.html

```diff
@@ -11,6 +11,70 @@ <h1>Amendments to the ECMAScript® 2021 Internationalization API Specification</
     </p>
   </emu-note>
 
+  <emu-clause id="sup-time-zone-names">
+    <h1><a href="https://tc39.es/ecma402/#sec-time-zone-names">Time Zone Names</a></h1>
+
+    <ins class="block">
+      <p>
+        <emu-xref href="#sec-time-zone-names"></emu-xref> defines a set of host-defined abstract operations concerning the names of supported time zones.
+        This section introduces additional requirements on how ECMAScript hosts that are conforming implementations of the ECMAScript 2021 Internationalization API Specification may implement these operations.
+      </p>
+    </ins>
+
+    <p>
+      The ECMAScript 2021 Internationalization API Specification identifies time zones using the Zone and Link names of the IANA Time Zone Database. Their canonical form is the corresponding Zone name in the casing used in the IANA Time Zone Database.
+    </p>
+
+    <p>
+      All registered Zone and Link names are allowed. <ins>Conforming</ins> implementations must recognize all such names, and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations. However, the set of combinations of time zone name and language tag for which localized time zone names are available is implementation dependent.
+    </p>
+
+    <emu-clause id="sec-intl-hostisvalidtimezonename">
+      <h1><a href="https://tc39.es/ecma402/#sec-isvalidtimezonename"><ins>Host</ins>IsValidTimeZoneName</a> ( _timeZone_ )</h1>
+
+      <p>
+        <del>The</del> <ins>Host</ins>IsValidTimeZoneName <ins>is a host-defined</ins> abstract operation <ins>which</ins> verifies that the _timeZone_ argument (which must be a String value) represents a valid Zone or Link name of the IANA Time Zone Database.
+        <ins>A conforming implementation must implement the abstract operation as follows:</ins>
+      </p>
+      <p>
+        The abstract operation returns true if _timeZone_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>, is equal to one of the Zone or Link names of the IANA Time Zone Database, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>. It returns false otherwise.
```

Comment:
Should the host be required to return true for a name forever, once it has reported true? What about vice versa?

Reply:
Consistent with the principle in https://tc39.es/proposal-temporal/#sec-temporal-tolocaltime note 2 would be to require that true remains always true, but not vice versa.